docs/Documentation/eagle_to_kestrel_transition.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Transitioning from Eagle to Kestrel
+---
+
+## Overview of steps
+
+Eagle is set to be decommissioned at the end of March, 2024. This page is meant to provide all necessary information to transition a project from Eagle to Kestrel. Transitioning a project can be broken down into [NUMBER] steps:
+
+1. Getting access to Kestrel
+2. Moving your files from Eagle to Kestrel
+3. Understanding the options for running your software on Kestrel
+
+    a. How to check if your software is available as a module on Kestrel
+
+    b. How to build your own software on Kestrel
+
+    c. General environment recommendations
+
+4. Submitting your jobs on Kestrel
+
+If you find yourself stuck on any of the above steps, please reach out to hpc-help@nrel.gov as soon as possible.
+
+## 1. Getting access to Kestrel
```

Comment:
I think this would be more clear as "Accessing Kestrel", since "getting access" is dependent on getting an allocation which isn't in the scope of this page.